tests/optimizers/tests_optimizers.py

```diff
@@ -101,10 +101,10 @@ def test_adagrad():
         #trainer.append_task(tasks.PrintVariable("Loss gradient: {}", loss.gradients[param]))
 
         # Monitor the gradient of `loss` w.r.t. to `param`.
-        gparam = tasks.MonitorVariable(loss.gradients[param])
-        trainer.append_task(gparam)
+        gparam = views.MonitorVariable(loss.gradients[param])
+        trainer._graph_updates.update(gparam.updates)
```

Comment:
But you'd probably want to access that monitored variable somehow, like you do below: `gparam.value`. If the tracker creates it, we would need to add a `value` property, like the old TrackVariable task had, so we can do `my_tracker.value`. Still we would need two lines, one to create the Tracker object and another to add it to the trainer.